ballerina/LICENSE

```diff
@@ -372,15 +373,15 @@ epl2           Eclipse Public License Version 2.0
                https://www.eclipse.org/legal/epl-2.0/
 cddl1          Common Development and Distribution License
                http://www.opensource.org/licenses/cddl1.php
-mit            MIT License
-               http://www.opensource.org/licenses/mit-license.php
 bsd            Berkeley License
                http://www.opensource.org/licenses/bsd-license.php
 bsd3           Berkeley License - 3
                http://opensource.org/licenses/BSD-3-Clause
+mit            MIT License
+               http://www.opensource.org/licenses/mit-license.php
 bouncy         Bouncy Castle License
                http://www.bouncycastle.org/licence.html
 edl1           Eclipse Distribution License Version 1.0
                http://www.eclipse.org/org/documents/edl-v10.php
 mpl10          Mozilla Public License Version 1.0
-               http://www.mozilla.org/MPL/
\ No newline at end of file
+               http://www.mozilla.org/MPL/
```

Comment:
Do we need these chnages? I think we can omit these

Reply:
Updated